cartridge/rpc.lua

```diff
@@ -279,7 +286,7 @@ local function call_remote(role_name, fn_name, args, opts)
         return errors.netbox_call(
             conn,
             '_G.__cluster_rpc_call_local',
-            {role_name, fn_name, args},
+            {role_name, fn_name, args, confapplier.get_instance_uuid()},
```

Comment:
Such approach adds new 36 bytes to each rpc call and makes it slower in general. Why can't we perform such check locally? Doesn't expelled instance know that it's actually expelled?